Waidhoferj/CSC-308-309-Project
# Art Submission

You can now submit new artwork to our app!

## Updates
- The `CreateArtworkMutation` now accepts a `pictures` attribute that is an array of images.
- The `CreateArtworkMutation` uses `foundBy` to add the artwork to the user's `personalPortfolio`
- From the camera page, users can't press the "go to art submission arrow" button on the right until they have taken one picture.
- Added styles to the ArtworkSubmission page
- Added logic to submit the artwork using `CreateArtworkMutation` 
- Handled navigation to the Artwork screen after user submits the form

## Known issues
- The screen doesn't play so nice with the TabBar component. Scrolling is a bit funky.
- No error handling for the mutation logic

Despite this, I think we should push forward and correct any issues after the demo (next quarter)

## Testing
1. Boot up frontend and backend servers
2. Go to the camera page.
2. Take a photo
3. Press the right arrow button onscreen on the camera page.
4. Fill out all of the fields on the Art Submission page
5. Press Post Art
6. Verify that you see an art page populated with the information that you submitted.
